### PR TITLE
Adds responsiveness to the navbar

### LINKED ISSUE #247: Create event button overflows in mobile devices
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
19264943_1338882936194999_277566138_n

**Expected behavior:**
screenshot from 2017-06-17 11-47-31

Working on the fix.

### PR DESCRIPTION
**Pull request by geekyd**
- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Deployment link:
https://open-event-frontend-app.herokuapp.com/

Fixes #247 
@niranjan94 @utkarshnath @CosmicCoder96 @harshitagupta30 @sumedh123 @anu-007 Please review.

### CODE REVIEW

File: app/templates/components/nav-bar.hbs [hunk 1]
```diff
@@ -54,12 +54,12 @@
   </div>
 </div>
 <div class="mobile only row">
-  <div class="ui navbar menu">
+  <div class="ui navbar {{if device.isMobile 'tiny'}} menu">
     <a class="menu item open sidebar">
-      <i class="large content icon"></i>
+      <i class="small content icon"></i>
```
Comment (CosmicCoder96) [inline app/templates/components/nav-bar.hbs]: Keep it just `content icon` don't make it small, it gets too small and I tested the navbar doesn't overflow without the small class.

File: app/templates/components/nav-bar.hbs [hunk 2]
```diff
@@ -68,7 +68,7 @@
         </a>
       {{/if}}
       <div class="item">
-        <a class="ui blue button" style="text-transform: uppercase; font-weight: 400;" href="{{href-to 'create'}}">
+        <a class="mini ui blue button button-text {{if device.isMobile 'small text'}}" style="text-transform: uppercase; font-weight: 400;" href="{{href-to 'create'}}">
```
Comment (CosmicCoder96) [inline app/templates/components/nav-bar.hbs]: Don't make the button mini, remove the mini class. I tested and it was fine. Small text class is enough for making it responsive across all portrait modes.

**Comment by anu-007** [comment]
image

see the screenshot above create event button still overflows

**Comment by anu-007** [comment]
image

see the screen shot above create event button still overflows

**Comment by geekyd** [comment]
@anu-007 Can you please specify the screen-size? Are you talking about standard screen size here? I have tested it for the standard screen sizes using the [this](https://chrome.google.com/webstore/detail/web-developer/bfbameneiokkgbdmiekhjnmfkcnldhhm) extension. Thanks

**Comment by anu-007** [comment]
tested on iPad (768x1024) from chrome dev tools

File: app/templates/components/nav-bar.hbs (hunk 2 above)
Comment (geekyd) [inline app/templates/components/nav-bar.hbs]: Actually the `mini` class does not work here. Removing it.

**Comment by geekyd** [comment]
@anu-007 Please check this 
Resolution :  (768x1024) 
screenshot from 2017-06-17 17-11-46

**Comment by geekyd** [comment]
@CosmicCoder96 Updated. Please review again. Thanks

**Comment by anu-007** [comment]
@geekyd checked again, worked fine (app is not updated previously).

**Comment by CosmicCoder96** [comment]
@geekyd Is the heroku link not updated?

**Comment by geekyd** [comment]
@CosmicCoder96 Yes.

**Comment by CosmicCoder96** [comment]
Please update it so I can have a final look before approving thanks :)

**Comment by geekyd** [comment]
@CosmicCoder96 It is updated try opening it in incognito mode.
https://open-event-frontend-app.herokuapp.com/

**Comment by CosmicCoder96** [comment]
@niranjan94 Please approve so that it can be merged thanks !

**Comment by geekyd** [comment]
@niranjan Please test & merge this. Thanks

**Pull request merged**